EduVPN/ConnectionService.swift

```diff
@@ -84,8 +84,9 @@ class ConnectionService: NSObject {
     /// - Returns: URL where config was installed
     /// - Throws: Error writing config to disk
     private func install(config: String) throws -> URL {
-        let tempDir = NSTemporaryDirectory()
-        let fileURL = URL(fileURLWithPath: tempDir + "/eduvpn.ovpn")
+        let tempDir = (NSTemporaryDirectory() as NSString).appendingPathComponent("org.eduvpn.app.temp") // Added .temp because .app lets the Finder show the folder as an app
```

Comment:
you might want to consider placing the `org.eduvpn.app.temp` in a more accessible location as reference (some configservice or something).